### DISCUSSION TITLE
Github student developer pack

### DISCUSSION THREAD

**Discussion by ghost** [community/community#131027] [GitHub Education]
Hello everyone, i can't get the Github student developer pack for unknown reasons. I already did all the requirements, but still getting rejected. Please help.
![Screenshot from 2024-07-04 20-38-52](https://github.com/community/community/assets/161864408/0a8e5dfe-b5e2-451d-a1ca-b8c642585865)
Some of the past rejections included:
![Screenshot from 2024-07-04 20-40-47](https://github.com/community/community/assets/161864408/1961f5ef-942f-49e5-a945-70a491badfa9)
![Screenshot from 2024-07-04 20-41-11](https://github.com/community/community/assets/161864408/e8fb3001-819f-4390-9fc7-d8be9d563ec5)

**Comment by calyxish** [comment]
I also experienced the same thing.
But they later verified mine.
All the messages under what happened should be given attention.
Make sure you provide all the things they need to verify based on the requirements expected from you.

Your name displayed on the GitHub profile should be the same as the one on your student ID or academic affiliate ID. The ID should be up to date.

For them to verify it should be the same dimension and picture format. they have stated when applying.

And also make sure you are verifying the exact name of your school. sometimes you can select the name of the school but a different location because of the location. Or maybe they are not able to fetch your details from your school's official website.

My Proves
![github pic 1](https://github.com/community/community/assets/148647663/48ca943a-68a0-48d5-b934-4b63669377c5)
![github pic 2](https://github.com/community/community/assets/148647663/050579fd-957f-4ef9-a011-9f0f9bd2cf70)
![github pic 3](https://github.com/community/community/assets/148647663/cf7f1cb2-11a7-4cf0-a6ab-f263a84dec18)

But never give up...
Press on till they verify you.

if I have answered you, check my comment as answered. if not comment on it again. lemme still be assisting you.

Thank you very much.
